thrift/src/main/java/com/linecorp/armeria/server/thrift/THttpServiceBuilder.java

```diff
@@ -178,7 +178,7 @@ public THttpService build() {
         @SuppressWarnings("UnstableApiUsage")
         final Map<String, List<Object>> implementations = Multimaps.asMap(implementationsBuilder.build());
 
-        final ThriftCallService tcs = ThriftCallService.of(implementations);
+        final ThriftCallService tcs = ThriftCallService.ofMultiple(implementations);
```

Comment:
There is erasure compilation error cause. So had to change the method name. Let me know if this is fine, or any better way to do this that I'm missing.